node-upstream-tests/node/run_tests.js

```diff
@@ -0,0 +1,273 @@
+'use strict'
+
+/* eslint-disable no-console */
+
+const childProcess = require('child_process')
+const path = require('path')
+const fsUtils = require('./fs_utils')
+
+const NODE_BIN = process.env['NODE_BIN'] || '/usr/bin/node'
+const NODE_REPO_PATH = process.env['NODE_REPO_PATH']
+if (NODE_REPO_PATH === undefined) {
+  throw new Error('The env variable NODE_REPO_PATH is not set. This is required to locate the root of the nodejs repo')
+}
+
+const MODULES = [
```

Comment:
Can the tests for each of these be run in parallel? Otherwise, is there any other way to reduce the amount of time needed to run the tests? Almost 5 minutes seems a bit excessive to run just a subset of the tests.

Reply:
Some of them can run in parallel, some of them should be run sequentially. They are in separate directories so it would be possible to differentiate them.

Reply:
Down to 1m30. Running more than 8 tests concurrently doesn't bring any improvement

Reply:
> Running more than 8 tests concurrently doesn't bring any improvement

That might be because they're running on a `small` instance, so maybe more parallelism could be achieved on a bigger machine. However, for now the current parallelism is fine 👍